src/sentry/static/sentry/app/views/settings/project/projectKeys/projectKeyDetails.jsx

```diff
@@ -481,7 +490,7 @@ export default class ProjectKeyDetails extends AsyncView {
     let features = new Set(project.features);
     let hasRateLimitsEnabled = features.has('rate-limits');
     let orgFeatures = new Set(organization.features);
-    let hasjsSdkLoaderEnabled = orgFeatures.has('relay');
+    let hasjsSdkLoaderEnabled = orgFeatures.has('js-loader');
```

Comment:
Does `relay` need to feed into something else in the ui now?

Reply:
At least not on this page but `relay` feature flag is used in other places.
This was here from the very first relay PR where we thought the "loader" is part of the relay.
It turned out the be it's own standalone thing now :)